### ISSUE TITLE
[Quest][Item] Gurf's Dignity

### ISSUE THREAD

**Issue by Xandriarn** [dalaranwow/dalaran-wow#4368]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: The item Gurf's Dignity dropped off a murloc on Azuremyst Isle. The tooltip states that it starts a quest. When right-clicking the item, I get an error message "You don't meet the requirements for that quest".

**Expected behaviour**: The right-click should open the quest text window and allow me to accept the quest. 

**Steps to reproduce the problem**:

1. Currently on quest "Murlocs... Why Here? Why Now?" to retrieve "Stillpine Grain" items from Stiltfin Murlocs. 
2. While killing murlocs, found the "Gurf's Dignity" item in my bag. I believe it dropped off of the named murloc Murgurgula.
3. Attempted to right-click the item to accept the quest.

**Include proofs for this behaviour**

The item is: http://www.wowhead.com/item=23850/gurfs-dignity

The quest it should give is: http://www.wowhead.com/quest=9564/gurfs-dignity

Further Info:

1) Once I went back and turned in the quest "Murlocs... Why Here? Why Now?" (http://www.wowhead.com/quest=9562/murlocs-why-here-why-now) the item then let me accept the quest.
2) Based on comments on wowhead for the item and quest, this is not expected behavior as others were able to accept the quest without turning in http://www.wowhead.com/quest=9562/murlocs-why-here-why-now

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
DB/Quest: Gurf's Dignity - Remove prev.quest: Murlocs... Why Here? Why Now?

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4368
* commit: 80dc4d9298b8a3759dac2a8782adec07d8537496

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
